tap_github/repository_streams.py

```diff
@@ -2022,7 +2022,7 @@ def http_headers(self) -> dict:
 
         Mock a web browser user-agent.
         """
-        return {"User-agent": "Mozilla/5.0"}
+        return {"User-Agent": "Mozilla/5.0"}
```

Comment:
Slightly off-topic for this PR, but GitHub [request](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#user-agent-required) that a clear app name be used, should this say `tap-github` instead?

Reply:
This is mocking user http requests, not pinging the API so I think we actually don't want to give a clear app name 😄

Reply:
Hmm, I thought this was pretending to be a web browser, but the stream is actually pinging the API, no? :thinking:

Reply:
No, this stream is a bit convoluted but it is actually pinging GitHub's website. The switch happens in `scrape_dependents` and we use beautifulsoup to parse the html

Reply:
Ok, I totally missed that. Just added a short comment to make it more obvious.